spec/system/comments/debates_spec.rb

```diff
@@ -84,6 +84,25 @@
     expect(page).not_to have_content grandchild_comment.body
   end
 
+  scenario "can collapse comments after adding a reply", :js do
+    parent_comment = create(:comment, body: "Main comment", commentable: debate)
+    create(:comment, body: "First subcomment", commentable: debate, parent: parent_comment)
+
+    login_as(user)
+    visit debate_path(debate)
+    within "#comment_#{parent_comment.id}_reply" do
```

Comment:
We usually leave a blank line before and after `within` blocks. Sorry to insist on this one :smile:.